tests/unit/plugins/test_ant_plugin.py

```diff
@@ -169,7 +170,9 @@ def test_get_build_environment_proxy(part_info, env, opts):
     properties = AntPlugin.properties_class.unmarshal({"source": "."})
     plugin = AntPlugin(properties=properties, part_info=part_info)
     with mock.patch.dict(os.environ, env):
-        assert plugin.get_build_environment() == {"ANT_OPTS": opts}
+        env = plugin.get_build_environment()
+        del env["JAVA_HOME"]
```

Comment:
instead of doing this I think we should just do `assert env["ANT_OPTS"] == opts`